### PR TITLE
Brain Atlas revision

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Comment by giovannivolpe** [comment]
@annacanal @mitemijalkov check all including documentation

File: braph2/atlas/BrainAtlas.m [hunk 1]
```diff
@@ -238,290 +256,296 @@ function setNotes(atlas, notes)
...
-%             if fid == -1, error('Cannot create JSON file'); end
-%             fwrite(fid, Json_structure, 'char');
-%             fclose(fid);
-%         end
+        function atlas = load_from_xls(varargin)
+            % LOAD_FROM_XLS loads brain atlas from XLS file
+            %
+            % ATLAS = LOAD_FROM_XLS('File', FILE) creates and initializes the
+            % brain atlas ATLAS by loading an XLS file ('*.xlsx' or '*.xls').
+            % It throws an error is the file does not exist
```
Comment (annacanal) [inline braph2/atlas/BrainAtlas.m:264]: Typo: It throws an error **if** the file does not exist

File: braph2/atlas/BrainAtlas.m [hunk 2]
```diff
@@ -238,290 +256,296 @@ function setNotes(atlas, notes)
...
+            tab = [
+                {atlas.getID(), {}, {}, {}, {}, {}};
+                table(br_ids, br_label, br_notes, br_x, br_y, br_z)
+                ];
+            
+            % save
+            writetable(tab, file, 'Sheet', 1, 'WriteVariableNames', 0);
+        end
+        function atlas = load_from_txt(varargin)
+            % LOAD_FROM_TXT loads brain atlas from XLS file
```
Comment (annacanal) [inline braph2/atlas/BrainAtlas.m:352]: % LOAD_FROM_TXT loads brain atlas from **TXT** file

File: braph2/atlas/BrainAtlas.m [hunk 3]
```diff
@@ -238,290 +256,296 @@ function setNotes(atlas, notes)
...
+                br_x = raw{i, 4};
+                br_y = raw{i, 5};
+                br_z = raw{i, 6};
+                br = BrainRegion(br_id, br_label, br_notes, br_x, br_y, br_z);
+                atlas.getBrainRegions().add(br_label, br);
+            end
+        end
+        function save_to_txt(atlas, varargin)
+            % SAVE_TO_TXT saves brain atlas to XLS file
+            %
```
Comment (annacanal) [inline braph2/atlas/BrainAtlas.m:400]: % SAVE_TO_TXT saves brain atlas to **TXT** file

File: braph2/atlas/BrainAtlas.m [hunk 4]
```diff
@@ -238,290 +256,296 @@ function setNotes(atlas, notes)
...
+                {atlas.getID(), {}, {}, {}, {}, {}};
+                table(br_ids, br_label, br_notes, br_x, br_y, br_z)
+                ];
+            
+            % save
+            writetable(tab, file, 'Delimiter', '\t', 'WriteVariableNames', 0);
+        end
+        function atlas = load_from_json(varargin)
+            % LOAD_FROM_JSON loads brain atlas from XLS file
+            %
```
Comment (annacanal) [inline braph2/atlas/BrainAtlas.m:445]: % LOAD_FROM_JSON loads brain atlas from **JSON** file

File: braph2/atlas/BrainAtlas.m [hunk 5]
```diff
@@ -238,290 +256,296 @@ function setNotes(atlas, notes)
...
+                br_x = intern_fields.x;
+                br_y = intern_fields.y;
+                br_z = intern_fields.z;
+                br = BrainRegion(br_id, br_label, br_notes, br_x, br_y, br_z);
+                atlas.getBrainRegions().add(br_label, br);
+            end
+        end
+        function save_to_json(atlas, varargin)
+            % SAVE_TO_JSON saves brain atlas to XLS file
+            %
```
Comment (annacanal) [inline braph2/atlas/BrainAtlas.m:499]: % SAVE_TO_JSON saves brain atlas to **JSON** file

File: braph2/atlas/PlotBrainSurf.m [hunk 6]
```diff
@@ -157,7 +157,7 @@
             end
             bs.settings = settings;
             
-            brain_surface_file = get_from_varargin('BrainMesh_ICBM152.nv' , 'BrainSurfaceType', varargin(:));
+            brain_surface_file = get_from_varargin('BrainMesh_ICBM152.nv' , 'BrainSurfaceType', varargin{:});
             bs.set_surface(brain_surface_file);
```
Comment (giovannivolpe) [inline braph2/atlas/PlotBrainSurf.m:161]: BrainSurfaceType -> BrainSurface

**Pull request merged**